### PR TITLE
August TC Newsletter - Initial Draft

### PR DESCRIPTION
**Pull request by alexander-hughes**
Signed-off-by: Alexander Hughes <Alexander.Hughes@pm.me>

### CODE REVIEW

File: src/pages/blog/airship-update-august-2020.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  - id: category-C98iZYrE1
+    label: Airship 2.0
+---
+
+What follows is the Airship Community Update for August 2020, brought to you by the Airship Technical Committee.
+<!-- more -->
+
+## **SPOTLIGHT ON...**
+
+- On 22-July at the [_OpenDev: Hardware Automation_](https://www.openstack.org/events/opendev-2020/opendev-schedule-2)
```
Comment (drewwalters96) [inline src/pages/blog/airship-update-august-2020.md:16]: Will everyone know what _the_ baremetal whitepaper is? A small preview of the whitepaper and its relevance to airship could go a long way. Something like:

_Several members of the Airship community contributed to a whitepaper recently released by the OpenStack Foundation that details the OpenDev community's investment in open source bare metal provisioning._

File: src/pages/blog/airship-update-august-2020.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- [*Pre-Alpha Airship Blog Series 4 - Shipyard - an Evolution of the Front Door*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-4-shipyard-an-evolution-of-the-front-door/)
+- [*Pre-Alpha Airship Blog Series 5 - Drydock and Its Relationship to Cluster API*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/)
+- [*Pre-Alpha Airship Blog Series 6 - Armada Growing Pains*](
+  https://www.airshipit.org/blog/pre-alpha-airship-blog-series-6-armada-growing-pains/)
+
+<br>
+
+Stay tuned for upcoming blog posts as Airship2.0 progresses through the beta milestone!
```
Comment (drewwalters96) [inline src/pages/blog/airship-update-august-2020.md:55]: Airship 2.0.

Comment (alexander-hughes) [reply src/pages/blog/airship-update-august-2020.md]: Done

File: src/pages/blog/airship-update-august-2020.md (hunk 1 above)
Comment (alexander-hughes) [reply src/pages/blog/airship-update-august-2020.md]: Done

**Pull request merged**